ci/build_cgmanifest.py

```diff
@@ -0,0 +1,190 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can this script be moved to a shared repository (eg: imx-iotcore) so that we only have to update it in one place rather than having to update multiple copies?

Reply:
I was debating that. I don't feel like there is a good answer. The way the pipelines work its definitively better to make the repo as self sufficient as possible (which means not relying on an external repo), but we already rely on it for build infrastructure...

I originally setup the script for the MSRsec repo which is fully independent of iotcore, and copied the paradigm to our other repos. Given that none of our other repos really make sense without iotcore I can do that quite easily.

Reply:
You could pull down the script using wget, the same way you pull the GCC tools.